presto-docs/src/main/sphinx/release/release-0.148.rst

```diff
@@ -13,37 +13,97 @@ General Changes
   ``ROWS BETWEEN UNBOUNDED PRECEDING AND 2 PRECEDING``
   would incorrectly use the first row as the window frame for the first two
   rows rather than using an empty frame.
-* Change default ``task.max-worker-threads`` to ``2`` times the number of cores.
+* Fix correctness issue when grouping on columns that are also arguments to aggregation functions.
+* Fix failure when chaining ``AT TIME ZONE``, e.g.
+  ``SELECT TIMESTAMP '2016-01-02 12:34:56' AT TIME ZONE 'America/Los_Angeles' AT TIME ZONE 'UTC'``.
+* Fix data duplication when ``task.writer-count`` configuration mismatches between coordinator and worker.
+* Fix bug where ``node-scheduler.max-pending-splits-per-node-per-task`` config is not always
+  honored by node scheduler. This bug could stop the cluster from making further progress.
 * Add ``colocated-joins-enabled`` to enable colocated joins by default for
   connectors that expose node-partitioned data.
 * Add support for ``REVOKE`` permission syntax.
-* Warn if Presto server is not using G1 garbage collector.
-* Improve planning of co-partitioned JOIN and UNION.
-* Improve planning of aggregations over partitioned data.
 * Add support for colocated unions.
-* Use HTTPS in JDBC driver when using port 443.
-* Increase default value for ``query.initial-hash-partitions`` to ``100``.
 * Add :func:`element_at` function for map type.
 * Add :func:`split_to_map` function.
+* Add :func:`zip` function.
 * Add ``ROW`` syntax for constructing row types.
+* Add support for `SMALLINT` and `TINYINT` types.
+* Add :func:`map_union` aggregation function.
+* Add re2j-td as backend for regular expression functions, this can be enabled via
```

Comment:
This item needs a lot more work to make it understandable by users. Something like "Add a new regex library based on RE2J that enables blah blah" and explain exactly how to enable and why you would want to.

Or remove it entirely if this feature is still experimental.